pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -877,6 +882,15 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 			}
 		}
 
+		canMountErr := volumeMounter.CanMount()
```

Comment:
I had that in mind previously but didn't change it because it affects the readability of code. (It's not immediately visible if the CanMount() is called in the second statement of the `if block` and for now since it's unlikely to cause a panic with only the nfs implementation.)

But in the future some other implementation of CanMount() can cause a panic, so I'll move it.